degiroapi/__init__.py

```diff
@@ -66,6 +72,16 @@ def login(self, username, password, totp=None):
 
         return client_info_response
 
+    def login_prompt(self, username=None, password=None, totp=None):
+        import getpass
+
+        if not username: username = input("Username: ")
+        if not password: password = getpass.getpass("Password:")
+        if not totp:     totp = getpass.getpass("totp (Leave empty if none):")
+        
+        if totp=="": return self.login(username, password)
+        else:        return self.login(username, password, totp)
```

Comment:
```suggestion
        return self.login(username, password, totp or None)
```

Same thing, more concise.

Reply:
Much better! I wasn't aware of that syntax.